spec/dummy/app/themes/default/assets/stylesheets/components/_currently-online.scss

```diff
@@ -1,8 +1,17 @@
 .currently-online {
-  @include position(fixed, null 0 0 null);
+  @include position(fixed, null 0 -1.25rem null);
   background-color: $base-border-color;
   padding: $base-spacing;
   width: rem(260);
+
+  &.is-expanded {
+    @include position(fixed, null 0 0 null);
+
+    .currently-online--users {
+      height: auto;
+      visibility: visible;
+    }
+  }
```

Comment:
I know this is ugly and I know this could use a little TLC in the form of some transitions or something but ... derp.